app/lib/reports/offline_session_exporter.rb

```diff
@@ -125,32 +125,42 @@ def columns
   end
 
   def patient_sessions
-    session
-      .patient_sessions
-      .includes(
-        patient: [
-          :consent_statuses,
-          :school,
-          { vaccination_records: %i[batch performed_by_user vaccine] }
-        ],
-        session: :programmes
-      )
-      .order_by_name
+    @patient_sessions ||=
+      session
+        .patient_sessions
+        .includes(
+          patient: [
+            :consent_statuses,
+            :school,
+            { parent_relationships: :parent },
+            {
+              vaccination_records: %i[
+                batch
+                performed_by_user
+                vaccine
+                programme
+                session
+              ]
+            }
+          ],
+          session: [{ programmes: :vaccines }, :location]
+        )
+        .order_by_name
   end
 
   def consents
     @consents ||=
       Consent
         .where(patient_id: patient_sessions.select(:patient_id))
         .not_invalidated
-        .includes(:parent, :patient)
+        .includes(:parent, patient: { parent_relationships: :parent })
```

Comment:
We also include the parent from the `Consent`, do you know why we also need to include it through the parent relationships? Is there some code elsewhere that could be updated to not need it and instead get the parent from the consent model?

Reply:
If we don't use `.includes(:parent, patient: { parent_relationships: :parent })` we get the following:

```
1. Count: 844 (91.8% of total)
   SELECT "parent_relationships".* FROM "parent_relationships" WHERE "parent_relationships"."patient_id" = N /*application='X'*/
```

I think I tracked down where these queries are coming from:

Starts here:

```
row[:health_question_answers] = Cell.new(
      health_question_answers(consents: grouped_consents),
      style: {
        alignment: {
          wrap_text: true
        }
      }
    )
```

then `health_question_answers` -> `ConsolidatedHealthAnswers` -> `HasHealthAnswers#who_responded` -> `Consent#parent_relationship`